invenio_communities/views/api.py

```diff
@@ -177,9 +181,21 @@ def get(self, community_id):
         community = Community.get(community_id)
         if not community:
             abort(404)
-        return self.make_response(
-                community, links_item_factory=default_links_item_factory)
 
+        etag = hashlib.sha1(
```

Comment:
`etag` is used to determine change in content of a resource at a given URL. Hence it's basically a `version_id`.